qemu/tests/qemu_guest_agent.py

```diff
@@ -126,11 +141,68 @@ def _check_ga_service(self, session, cmd_check_status):
         :param session: use for sending cmd
         :param cmd_check_status: cmd to check if ga service is started
         '''
-        error_context.context("Check whether qemu-ga service is started.", logging.info)
+        error_context.context("Check whether qemu-ga service is started.",
+                              logging.info)
         s, o = session.cmd_status_output(cmd_check_status)
         return s == 0
 
     @error_context.context_aware
+    def _get_latest_pkg(self):
+        """
+        get latest qemu-guest-agent rpm package.
+        :return: rpm pkg list
+        """
+        virt_module_stream = self.params.get("virt_module_stream", "")
+        guest_name = self.params.get("guest_name")
+        download_root = self.params["download_root_url"]
+        query_timeout = 180
+
+        error_context.context("Check if brew command is presented.",
+                              logging.info)
+        output = process.system_output("which brew").decode()
+        if re.search(r"no brew", output, re.I):
+            self.test.error("Pls install brew command in host.")
+
+        error_context.context("Get latest virt module tag of %s"
+                              " stream." % virt_module_stream,
+                              logging.info)
+        # target release,such as 810,811
+        target_release = re.findall(r'rhel(\d+)-\d+', guest_name, re.I)[0]
+        # get tag pattern,such as module-virt-8.1-80101xxxxx
+        if virt_module_stream == "rhel":
+            # for slow train,didn't know 810 or 811.
+            # module-virt-rhel-801xxx
+            target_release = target_release[:-1]
+        tag_version = "0".join(target_release)
+        # module-virt-8.1-80101 or module-virt-rhel-801
+        platform_tag = "module-virt-%s-%s" % (virt_module_stream,
+                                              tag_version)
+        get_latest_mdl_tag_cmd = "brew list-targets |grep"
+        get_latest_mdl_tag_cmd += " %s |sort -r |head -n 1" % platform_tag
+        latest_mdl_tag = process.system_output(get_latest_mdl_tag_cmd,
+                                               shell=True,
+                                               timeout=query_timeout
+                                               ).strip().split()[0].decode()
+        error_context.context("Get qemu-kvm build name via tag.",
+                              logging.info)
+        get_build_name_cmd = "brew list-tagged %s" % latest_mdl_tag
```

Comment:
`brew --quiet --topdir="%s" list-tagged ${latest_mdl_tag} --path --arch=${arch} | grep qemu-guest-agent`, you can use this command to get the full url

Reply:
Good suggestion.Thanks.
Updated the code.